### PR TITLE
Add 11.31 release notes

### PR DESCRIPTION
**Pull request by tas50**
#### Description

New week. New release

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [ ] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)
+
+---
+
+## 🎉 NEW FEATURES
+
+### Explore query packs results space-wide
+
+Understand the configuration of systems throughout your spaces with query packs. Query pack pages now expose results returned for each asset. Use Mondoo's two-dozen out of the box query packs to expose important configuration data such as running processes or available memory. If those query packs don't include quite what you need, write your own query packs to expose business specific asset configuration data throughout your spaces.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:20]:
```suggestion
Understand the configuration of systems throughout your spaces with query packs. Query pack pages now expose results returned for each asset. Use Mondoo's two dozen out-of-the-box query packs to expose important configuration data such as running processes or available memory. If those query packs don't include quite what you need, write your own query packs to expose business-specific asset configuration data throughout your spaces.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+![Space wide asset queries](/img/releases/2024-11-19-mondoo-11.31-is-out/query_pack_assets.png)
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:22]:
```suggestion
![Space-wide asset queries](/img/releases/2024-11-19-mondoo-11.31-is-out/query_pack_assets.png)
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+### New CIS NGINX benchmark policies
+
+Keep critical web servers, proxy servers, and load balancers secure with new CIS NGINX benchmarks. These six policies include 91 checks in total designed to ensure that NGINX is not only installed securely but configured to protect sensitive data from prying eyes.
+
+![NGINX policy page](/img/releases/2024-11-19-mondoo-11.31-is-out/nginx_policy.png)
+
+### Run Mondoo in Azure CloudShell
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:30]:
```suggestion
### Run Mondoo in Azure Cloud Shell
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+Secure your Azure subscriptions without setting up integrations or copying credentials to your local workstation. Now Mondoo can install and run directly in Azure CloudShell. Follow the Security Workstation settings to start scanning in just minutes.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:32]:
```suggestion
Secure your Azure subscriptions without setting up integrations or copying credentials to your local workstation. Now Mondoo can install and run directly in Azure Cloud Shell. Follow the Security Workstation settings to start scanning in just minutes.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+![Azure CloudShell Installation](/img/releases/2024-11-19-mondoo-11.31-is-out/cloudshell_install.png)
+
+## 🧹 IMPROVEMENTS
+
+### Priority findings on the spaces page
+
+Quickly understand the spaces that need your attention the most with priority findings shown on the spaces page. Priority findings is the total of all critical and high findings on the space, helping you to better prioritize your effort.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:40]:
```suggestion
Quickly understand the spaces that need your attention the most with a count of priority findings shown on the spaces page. The number of priority findings is the total of all critical and high findings on the space. Mondoo highlights them to help you to better prioritize your effort.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+![Spaces page tiles](/img/releases/2024-11-19-mondoo-11.31-is-out/priority_findings.png)
+
+### Faster load times in compliance
+
+Track your space compliance quicker than ever, with up to 10x faster load times in the Compliance Frameworks page.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:46]:
```suggestion
Track your compliance quicker than ever, with up to 10x faster load times in the Compliance Frameworks page.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+- New `selectiveCodeOwnerRemovals` field
+
+#### gitlab.project.member
+
+- New `state` field
+- New `username` field
+
+## 🐛 BUG FIXES AND UPDATES
+
+- Improve rendering of descriptions and remediation steps in Mondoo Linux Security policy.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:78]:
```suggestion
- Improve the rendering of descriptions and remediation steps in the Mondoo Linux Security policy.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+- Move all service checks in the Mondoo Linux Security policy into a Sensitive Services chapter.
+- Expand the Mondoo Linux Security policy `Ensure FTP servers are stopped and not enabled` check to include Pure-FTPd, and ProFTPD.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:80]:
```suggestion
- Expand the Mondoo Linux Security policy `Ensure FTP servers are stopped and not enabled` check to include Pure-FTPd and ProFTPD.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+- Expand the Mondoo Linux Security policy `Ensure IMAP and POP3 servers are stopped and not enabled` check to include the Cyrus IMAP server.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:81]:
```suggestion
- Expand the Mondoo Linux Security policy's `Ensure IMAP and POP3 servers are stopped and not enabled` check to include the Cyrus IMAP server.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+- Expand the Mondoo Linux Security policy `Ensure DNS server is stopped and not enabled`, `Ensure NIS server is stopped and not enabled`, and `Ensure talk server is stopped and not enabled` checks to support Debian and Ubuntu based systems.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:82]:
```suggestion
- Expand the Mondoo Linux Security policy's `Ensure DNS server is stopped and not enabled`, `Ensure NIS server is stopped and not enabled`, and `Ensure talk server is stopped and not enabled` checks to support Debian- and Ubuntu-based systems.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+- Add remediation steps for SLES/openSUSE to the Mondoo Linux Security policy.
+- Fix the Mondoo Linux Security policy's `Ensure SSH Protocol is set to 2` check incorrectly running on Ubuntu 24.04 systems.
+- Fix the Mondoo Linux Security policy's `Ensure mail transfer agent is configured for local-only mode` incorrectly identifying the IPv6 loopback address.
+- Ensure remediation steps in the Mondoo Linux Security policy include service restarts where necessary.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:86]:
```suggestion
- Ensure that the remediation steps in the Mondoo Linux Security policy include service restarts where necessary.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+
+### New CIS NGINX benchmark policies
+
+Keep critical web servers, proxy servers, and load balancers secure with new CIS NGINX benchmarks. These six policies include 91 checks in total designed to ensure that NGINX is not only installed securely but configured to protect sensitive data from prying eyes.
+
+![NGINX policy page](/img/releases/2024-11-19-mondoo-11.31-is-out/nginx_policy.png)
+
+### Run Mondoo in Azure Cloud Shell
+
+Secure your Azure subscriptions without setting up integrations or copying credentials to your local workstation. Now Mondoo can install and run directly in Azure Cloud Shell. Follow the Security Workstation settings to start scanning in just minutes.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:32]:
```suggestion
Secure your Azure subscriptions without complex Entra ID and Azure configuration. Now Mondoo can install and run directly in Azure Cloud Shell: Simply paste the Workstation Quick Setup command in Azure Cloud Shell.
```

Comment (misterpantz) [reply releases/2024-11-19-mondoo-11.31-is-out.md:32]: I dunno if that's right.

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+
+### New CIS NGINX benchmark policies
+
+Keep critical web servers, proxy servers, and load balancers secure with new CIS NGINX benchmarks. These six policies include 91 checks in total designed to ensure that NGINX is not only installed securely but configured to protect sensitive data from prying eyes.
+
+![NGINX policy page](/img/releases/2024-11-19-mondoo-11.31-is-out/nginx_policy.png)
+
+### Run Mondoo in Azure Cloud Shell
+
+Secure your Azure subscriptions without complex Entra ID and Azure configuration. Now Mondoo can install and run directly in Azure Cloud Shell: Simply paste the Workstation Quick Setup command in Azure Cloud Shell.
```
Comment (tas50) [inline releases/2024-11-19-mondoo-11.31-is-out.md:32]:
```suggestion
Secure your Azure subscriptions without complex Azure credential configuration. Now Mondoo can install and run directly in Azure Cloud Shell: Simply paste the Workstation Quick Setup command in Azure Cloud Shell.
```

File: releases/2024-11-19-mondoo-11.31-is-out.md [hunk 14]
```diff
@@ -0,0 +1,90 @@
...
+- Expand the Mondoo Linux Security policy's `Ensure DNS server is stopped and not enabled`, `Ensure NIS server is stopped and not enabled`, and `Ensure talk server is stopped and not enabled` checks to support Debian- and Ubuntu-based systems.
+- Add remediation steps for SLES/openSUSE to the Mondoo Linux Security policy.
+- Fix the Mondoo Linux Security policy's `Ensure SSH Protocol is set to 2` check incorrectly running on Ubuntu 24.04 systems.
+- Fix the Mondoo Linux Security policy's `Ensure mail transfer agent is configured for local-only mode` incorrectly identifying the IPv6 loopback address.
+- Ensure that the remediation steps in the Mondoo Linux Security policy include service restarts where necessary.
+- Fix failing compliance reports in some spaces.
+- Fix some AWS and Azure platforms not displaying correctly in the inventory overview.
+- Fix the link to the CloudFormation template during AWS Serverless integration setup with non-us-east-1 regions.
+- Support searching for software, compliance framework, compliance controls in space search.
```
Comment (misterpantz) [inline releases/2024-11-19-mondoo-11.31-is-out.md:90]:
```suggestion
- Add support for searching for software, compliance framework, and compliance controls in a space.
```

**Review by misterpantz — APPROVED** [review]

**Pull request merged**